projects/netgrif-components-core/src/lib/resources/engine-endpoint/public/public-task-resource.service.ts

```diff
@@ -9,7 +9,6 @@ import {filter, map} from 'rxjs/operators';
 import {FilterType} from '../../../filter/models/filter-type';
 import {Filter} from '../../../filter/models/filter';
 import {Page} from '../../interface/page';
-import {ChangedFieldContainer} from '../../interface/changed-field-container';
```

Comment:
these changes are also from https://github.com/netgrif/components/pull/86

Reply:
Yes, they are, as I mentioned blocking PR #86 in this PR's description. I created this branch from NAE-1680 to avoid incorrect functioning of taskRef.

Reply:
my bad. I didn't notice that this PR has some blockers. Sorry